lib/src/component/suspense_component.dart

```diff
@@ -67,11 +67,11 @@ mixin SuspensePropsMixin on UiProps {
   /// The actual UI you intend to render. If children suspends while rendering, the Suspense boundary will
   /// switch to rendering fallback.
   @override
-  /*ReactNode*/ List<dynamic>? get children;
+  List<ReactNode>? get children;
```

Comment:
Similar to above, the changes to props in this component are technically breaking changes, but I think we could get away with these in particular since, by comparison, there should be almost no components that read the children/fallback props from this props mixin.